demo/web/index.html

```diff
@@ -0,0 +1,192 @@
+                let i = Math.trunc(f * INT16_MAX);
+                if (f > INT16_MAX) i = INT16_MAX;
+                if (f < INT16_MIN) i = INT16_MIN;
+                return i;
+              })
+            );
+
+            writeMessage("Diarizing audio file...");
+            const { segments } = await falcon.process(i16PCM, {
+              transfer: true,
+            });
+            setSegmentsTable(segments);
+            writeMessage("Diarizing audio file... done!");
+          });
+        });
+
+        const displayTimer = document.getElementById("displayTimer");
+        const recordButton = document.getElementById("recordAudio");
+        const stopRecord = document.getElementById("stopRecord");
+
+        let timer = null;
+        let currentTimer = 0.0;
+        let audioData = [];
+        const recorderEngine = {
+          onmessage: (event) => {
+            switch (event.data.command) {
+              case "process":
+                audioData.push(event.data.inputFrame);
+                break;
+            }
+          },
+        };
+        recordButton.addEventListener("click", async () => {
+          displayTimer.style.display = "inline";
+          stopRecord.style.display = "inline";
+          recordButton.style.display = "none";
+
+          currentTimer = 0.0;
+          audioData = [];
+          try {
+            writeMessage("Recording audio...");
+            await window.WebVoiceProcessor.WebVoiceProcessor.subscribe(
+              recorderEngine
+            );
+            timer = setInterval(() => {
+              currentTimer += 0.1;
+              displayTimer.innerText = `${currentTimer.toFixed(1)} / 120`;
+              if (currentTimer === 120) {
+                stopRecord.click();
+              }
+            }, 100);
+          } catch (e) {
+            writeMessage(e);
+          }
+        });
+
+        stopRecord.addEventListener("click", async () => {
+          displayTimer.style.display = "none";
+          stopRecord.style.display = "none";
+          recordButton.style.display = "inline";
+
+          await window.WebVoiceProcessor.WebVoiceProcessor.unsubscribe(
+            recorderEngine
+          );
+          clearInterval(timer);
+
+          const frames = new Int16Array(audioData.length * 512);
+          for (let i = 0; i < audioData.length; i++) {
+            frames.set(audioData[i], i * 512);
+          }
+
+          writeMessage("Diarizing audio file...");
+          const { segments } = await falcon.process(frames, {
+            transfer: true,
+          });
+          setSegmentsTable(segments);
+          writeMessage("Diarizing audio file... done!");
+        });
+      };
+
+      function writeMessage(message) {
+        console.log(message);
+        document.getElementById("status").innerHTML = message;
+      }
+
+      function setSegmentsTable(segments) {
+        let html = `
+          <tr>
+            <th>startSec</th>
```

Comment:
```suggestion
            <th>Start time (s)</th>
```
For showing the output, can we have a table similar to octopus web demo: https://github.com/Picovoice/octopus/blob/main/demo/web/index.html